src/PJ_sch.c

```diff
@@ -0,0 +1,206 @@
+/*
+** Author : Piyush Agram
+** This code is in the public domain and implements the SCH coordinate system first developed at JPL.
```

Comment:
I'm making this contribution as a volunteer (in my own free time). The appropriate documentation for the SCH coordinate system has been officially approved for public release. I'm happy to change the headers to read something like the example rouault pointed me to.